pir/cpp/benchmark.cpp

```diff
@@ -7,7 +7,8 @@
 
 namespace pir {
 
-uint32_t dimensions = 2;
+constexpr std::size_t ITEM_SIZE = 0;
```

Comment:
Why zero?

Reply:
Zero indicates that each item is one plaintext, whatever maximum size is allowed. This makes it easy to use the integer encoder for our previous tests.